### PR TITLE
Courtney/feature/navbar

### PR DESCRIPTION
**Pull request by Courtney-Lippman**
- [ ]  Wrote Tests 
- [x] Implemented 
- [x] Reviewed

## Necessary checkmarks:

- [x] All Tests are Passing in all environments

- [x] The code will run locally

## Type of change

- [x] New feature
- [ ] Bug Fix
- [ ] Refactor
- [x] Styling
- [ ] Testing
- [ ] This change requires a documentation update

## Description of Change:
This Change includes adding elements and styling to the Navbar component.
Issue: #12 #11 

(note that the other page names are still listed at the bottom of the browser. I left them there to give context to the other developers on where the Navbar is in relation to the other pages.)

## Check the correct boxes

- [x]  This broke nothing
- [ ]  This broke some stuff
- [ ]  This broke everything

## Checklist:

- [ ]  My code has no unused/commented out code
- [x]  My code follows the style guidelines of this project
- [x]  My code does not generate new warnings
- [x]  I have reviewed my code
- [ ]  I have commented my code, particularly in hard-to-understand areas (if applicable)

### CODE REVIEW

File: src/Components/NavBar/NavBar.js [hunk 1]
```diff
@@ -1,10 +1,64 @@
...
+          <p>Home</p>
+        </NavLink>
+        <NavLink 
+        className="watchlist-button" 
+        id='watchlist-button' 
+        to='/watchlist'
+        >
+          <p>My Watchlist</p>
+        </NavLink>
+      </section>
```
Comment (sam-rice) [inline src/Components/NavBar/NavBar.js]: Nice use of `<NavLink>`s! I don't have much experience with them, so I looked up the docs on them (see below). One thought is, have you thought about swapping the `<section>` element containing the `<NavLink>`s for a `<nav>` element and using an unordered list? Just thinking about accessibility. 

This is from the React docs on `<NavLink>`:
Screen Shot 2023-02-07 at 6 13 35 PM

File: src/Components/NavBar/_NavBar.scss [hunk 2]
```diff
@@ -0,0 +1,49 @@
+$nav-background-color:#101419;
+$text-color:  #D9D9D9;
+
+@mixin centerColumn() {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
```
Comment (sam-rice) [inline src/Components/NavBar/_NavBar.scss:8]: Putting Sass to work right away!

Comment (Courtney-Lippman) [reply src/Components/NavBar/_NavBar.scss:8]: Haha @mixins are amazing!

File: src/Components/NavBar/NavBar.js (hunk 1 above)
Comment (Courtney-Lippman) [reply src/Components/NavBar/NavBar.js]: Hmm.... That is a really good suggestion! Let me try out the above!

Comment (Courtney-Lippman) [reply src/Components/NavBar/NavBar.js]: @sam-rice I changed the section to a nav element and wrapped the search input and nav links in a ul as li elements. Let me know what you think!

Comment (sam-rice) [reply src/Components/NavBar/NavBar.js]: Looks perfect to me!

**Pull request merged**